DOCS-9758-sftp-copy-move-example-kt (#1235)
mulesoft/docs-connectors: sftp/1.3/modules/ROOT/pages/sftp-connection-dynamically.adoc

The following example shows how to configure SFTP Connector to connect dynamically to multiple servers:

. An HTTP Listener source initiates the flow by reading random content posted via HTTP.
. The File Connector Read operation loads a recipients CSV file that contains a random set of SFTP destinations, with columns such as host, user, and port.
. A DataWeave transformation maps the content and splits the CSV file.

> . A For Each component and an SFTP Write operation write the contents into each of the SFTP destinations. On each For Each iteration, the expressions set in the SFTP global configuration element resolves to a different value and establish different connections to each of the servers.

Reviewer: ```suggestion
. A For Each component and an SFTP Write operation write the contents into each of the SFTP destinations. On every For Each iteration, the expressions set in the SFTP global configuration element resolve to a different value and establish different connections to each of the servers.
```

> To accomplish this example, create the Mule application, configure HTTP, File and SFTP global configuration elements, and run and test the application with curl commands:

Reviewer: ```suggestion
To accomplish this example, create the Mule application, configure HTTP, File, and SFTP global configuration elements, then run and test the application with curl commands:
```

+
. Drag a *For Each* component to the right of the *Transform Message* component.
. Drag an *SFTP Write* operation into the *For Each* component. +

> On each For Each iteration, the *Write* operation writes the contents into each of the SFTP destinations.

Reviewer: ```suggestion
For every For Each iteration, the *Write* operation writes the contents into each of the SFTP destinations.
```

=== Run and Test your Mule Application

After creating your Mule application, run and test it:

. In Studio, save your Mule app.

> . Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*. +.

Reviewer: ```suggestion
. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*. 
```
